packages/services/src/manager.ts

```diff
@@ -221,6 +230,11 @@ export namespace ServiceManager {
      */
     readonly terminals: Terminal.IManager;
 
+    /**
+     * The user manager for the manager.
+     */
+    readonly user: UserManager.IManager;
```

Comment:
Shouldn't this be optional to be backward compatible?

Reply:
I'll do that when backporting #13428